ext/configuration.h

```diff
@@ -109,6 +109,7 @@ enum ddtrace_dbm_propagation_mode {
     CONFIG(STRING, DD_SPAN_SAMPLING_RULES_FILE, "", .ini_change = ddtrace_alter_sampling_rules_file_config)    \
     CONFIG(SET_LOWERCASE, DD_TRACE_HEADER_TAGS, "")                                                            \
     CONFIG(INT, DD_TRACE_X_DATADOG_TAGS_MAX_LENGTH, "512")                                                     \
+    CONFIG(SET, DD_TRACE_PEER_SERVICE_MAPPING, "")                                                             \
```

Comment:
We have a config type MAP (instead of SET) which will split comma separated key:value into a key/value array. This seems more suitable here than doing string parsing to the next colon in iteration later.

Reply:
One thing, though, is that the `peer_service_wrong_values.phpt` test would now fail because of the wrong formatting `foo=bar,only_tag,foo:bar` (the last `foo:bar` won't get interpreted). If this is what we want, then great, else I would need to change `zai_config_decode_map`.

Reply:
`bar,only_tag,foo:bar` will be parsed as `bar,only_tag,foo` key, `bar` value, right?

But I don't care too much about that case, it's wrong, it should give some not totally wrong value and not crash.

Reply:
> bar,only_tag,foo:bar will be parsed as bar,only_tag,foo key, bar value, right?

Well, yes indeed... See that way...

Alright then :+1: